src/documents/firstDiscoveryEditor.md

```diff
@@ -0,0 +1,385 @@
+        panelTotalNum: "{firstDiscoveryEditor}.panels.length"
+    }
+}</code></pre>
+        </td>
+    </tr>
+    <tr>
+        <td>`messageLoader`</td>
+        <td>Specifies the component and means for loading in the JSON message bundle files.</td>
+        <td>[`"fluid.prefs.resourceLoader"`](http://docs.fluidproject.org/infusion/development/LocalizationInThePreferencesFramework.html)</td>
+        <td>
+        <pre><code>messageLoader: {
+    options: {
+        locale: "{prefsEditorLoader}.settings.preferences.gpii_firstDiscovery_language"
+    }
+}</code></pre>
+        </td>
+    </tr>
+</table>
+
+## Options
+
+This component can be configured using the following
+[options](http://docs.fluidproject.org/infusion/development/ComponentOptionsAndDefaults.html):
+
+<table>
+    <tr><th>Name</th><th>Description</th><th>Values</th><th>Default</th></tr>
+    <tr>
+        <td>`defaultLocale`</td>
+        <td>The language code to use, for selecting the correct message bundle, when no language has been provided in the preferences.</td>
+        <td>A valid language code such as: `"en-US"`, `"fr-FR"`, `"es-MX"`, `"de-DE"`, `"nl-NL"`, `"sv-SE"`</td>
+        <td>
+        This resolves to "en-US":<br/>
+        <pre><code>defaultLocale: {
+    expander: {
+        funcName: "fluid.get",
+        args: [{
+            expander: {
+                funcName: "fluid.defaults",
+                args: ["gpii.firstDiscovery.schemas.language"]
+            }
+        }, ["schema", "properties", "gpii.firstDiscovery.language", "default"]]
+    }
+}</code></pre>
+        </td>
+    </tr>
+    <tr>
+        <td>`tooltipOptions`</td>
+        <td>The options to be used by any descendant tooltip components.</td>
+        <td></td>
+        <td>
+        <pre><codetooltipOptions: {
+    delay: 0,
+    duration: 0,
+    position: {
+        my: "left bottom",
+        at: "right+1 top"
+    },
+    styles: {
+        tooltip: "gpii-fd-tooltip"
+    },
+    items: ".gpiic-fd-tooltip:not([disabled])"
+}</code></pre>
+        </td>
+    </tr>
+    <tr>
+        <td>`selectors`</td>
+        <td>Javascript object containing selectors for various fragments of the markup, including the containers for the subcomponents.</td>
+        <td></td>
+        <td>See [Selectors](#selectors)</td>
+    </tr>
+    <tr>
+        <td>`styles`</td>
+        <td>Specific class names used to achieve the look and feel.</td>
+        <td></td>
+        <td>
+        This resolves to "en-US":<br/>
+        <pre><code>styles: {
+    active: "gpii-fd-active",
+    show: "gpii-fd-show",
+    currentPanel: "gpii-fd-current"
+}</code></pre>
+        </td>
+    </tr>
+    <tr>
+        <td>`modelListeners`</td>
+        <td>JavaScript object containing model paths and the listeners that are activated when changes happen to those paths.</td>
+        <td>Keys in the object are event names, values are functions or arrays of functions.</td>
+        <td>See [Model](#model)</td>
```

Comment:
All the same, it should also link to the relevant section of our docs which describes the layout and purpose of the modelListeners block - which is http://docs.fluidproject.org/infusion/development/ChangeApplierAPI.html#registering-interest-in-model-changes-using-a-changeapplier